packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership.jsx

```diff
@@ -38,22 +35,21 @@ export const ProofOfOwnership = ({
                 };
             }
         });
-        return { groups, total_documents_required };
+        return { groups };
     }, [cards, is_dark_mode]);
     useEffect(() => {
         setStatus(account_status?.authentication?.ownership?.status?.toLowerCase());
     }, [account_status]);
     const onTryAgain = () => {
         setStatus(POO_STATUSES.none);
     };
-    if (needs_verification && status !== POO_STATUSES.rejected) {
+    if (cards?.length > 0 && status !== POO_STATUSES.rejected) {
         return (
             <ProofOfOwnershipForm
                 grouped_payment_method_data={grouped_payment_method_data.groups}
                 updateAccountStatus={updateAccountStatus}
                 refreshNotifications={refreshNotifications}
                 client_email={client_email}
-                total_documents_required={grouped_payment_method_data.total_documents_required}
             />
         ); // Proof of ownership is required.
     } else if (status === POO_STATUSES.verified) {
```

Comment:
@jim-deriv do you get a warning here 'unnecessary else after return'? i'd prefer to improve this especially if the task scope allows🙂